docker-wrappers/OmicsIntegrator2/Dockerfile

```diff
@@ -2,5 +2,14 @@
 # https://github.com/fraenkel-lab/OmicsIntegrator2
 FROM continuumio/miniconda3:4.9.2
 
+COPY 0001-disable-graph-exports.patch .
+
+RUN git clone https://github.com/agitter/OmicsIntegrator2 && \
+    cd OmicsIntegrator2 && \
+    git reset --hard 568f170eae388e42e923c478ac9f3308b487760b && \
+    git config user.email "email@example.com" && \
+    git config user.name "Non-existent User" && \
```

Comment:
git has to set a user before applying a patch? We don't have a SPRAS bot set up, so I can't think of a better default email and user to use. This is all inside an image and not committed so it shouldn't matter, right?

Reply:
Indeed. You can also just apply patch files directly, but this seemed like an okay nice tradeoff to get the niceties of `git patch`.